thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/anomaly/monitor/MonitorTaskRunner.java

```diff
@@ -108,6 +111,28 @@ private void executeMonitorUpdate(MonitorTaskInfo monitorTaskInfo) {
     }
   }
 
+  private int cleanUpMergedAnomalies(int cleanUpDays) {
+    DateTime cleanupStartDate = new DateTime().withTimeAtStartOfDay().minusDays(cleanUpDays);
+    DateTime cleanupEndDate = cleanupStartDate.plusDays(1);
```

Comment:
Also, how often the monitoring job runs? If it's running daily and failing on someday, there might still be some anomalies left to be not cleaned. 

Can we consider the following options?
- Instead of plus 1 day, we can add some additional lookback to make it more robust.
- Get the last successful monitoring job, and clean up from that time till the 90 days before.

Reply:
Good points. 
By looking at the last successful monitoring job is also not robust since this clean may fail as well.
Will add more buffer but still want to make it lightweight.